mlflow/tensorflow/callback.py

```diff
@@ -0,0 +1,102 @@
+from tensorflow import keras
+
+from mlflow import log_param, log_text
+from mlflow.utils.autologging_utils import BatchMetricsLogger
+
+
+class MLflowLoggingCallback(keras.callbacks.Callback):
```

Comment:
```suggestion
class MLflowTensorflowCallback(keras.callbacks.Callback):
```

?

Reply:
or

```suggestion
class MLflowCallback(keras.callbacks.Callback):
```

? wandb uses this convention: https://docs.wandb.ai/guides/integrations/xgboost#get-started

Reply:
or

```suggestion
class TensorflowCallback(keras.callbacks.Callback):
```